src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -79,6 +79,14 @@ namespace AppInstaller::Utility
                 return m_currentBrk;
             }
 
+            UChar32 CodePointAt(int32_t offset)

```

Comment:
>CodePointAt [](start = 20, length = 11)

Just from an API design standpoint, I would think it better for this to be `UChar32 CurrentCodePoint() const` and to have a free function `UChar32 CodePointAt(std::string_view, int32_t)` [you don't actually need this second one, I'm just pointing out how I would expect it to be designed]